x-pack/solutions/security/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/insights/results/policy_response_failure.tsx

```diff
@@ -32,31 +32,20 @@ export const WorkflowInsightsPolicyResponseFailureResult = ({
   const { ariaLabel, actionText, expandMessage, collapseMessage } =
     WORKFLOW_INSIGHTS.issues.remediationButton.policyResponseFailure;
 
-  const { euiTheme } = useEuiTheme();
-
-  const [isExpanded, setIsExpanded] = useState(false);
-  const [showViewMore, setShowViewMore] = useState(false);
-  const textRef = useRef<HTMLDivElement>(null);
-
-  useEffect(() => {
-    const textElement = textRef.current;
-    if (!textElement) return;
-
-    const clone = textElement.cloneNode(true) as HTMLDivElement;
-    clone.style.cssText = `
-    position: absolute;
-    visibility: hidden;
-    height: auto;
-    width: ${textElement.offsetWidth}px;
-    -webkit-line-clamp: unset;
-    display: block;
-  `;
-
-    document.body.appendChild(clone);
-    setShowViewMore(clone.offsetHeight > textElement.offsetHeight);
-    document.body.removeChild(clone);
+  const parsedRemediation = useMemo(() => {
+    try {
+      // remove escape characters (e.g. C:\\file\\path -> C:\file\path)
+      return JSON.parse(`"${insight.remediation.descriptive ?? ''}"`);
```

Comment:
Is this actually unescaping JSON escaped strings, or just fixing double backslashes in file paths?
If the latter how about a simple 
```typescript
const parsedRemediation = insight.remediation.descriptive?.replace(/\\\\/g, '\\') ?? '';
```